### ISSUE TITLE
Add effect sizes for hypothesis tests

### ISSUE THREAD

**Issue by danlwarren** [danlwarren/ENMTools#159]
Would be nice to report effect sizes, not just p values.  We'll need to take into account that these are possibly highly non-normal, bounded distributions though.

**Comment by danlwarren** [comment]
I've been looking into this this afternoon and it seems like we might want to follow Grissom and Kim 2012.  They suggest using U (from Mann-Whitney U test) and dividing by the product of the two sample sizes, which in our case would just be the number of Monte Carlo replicates.

![image](https://user-images.githubusercontent.com/3770891/85366631-b5d0bd80-b562-11ea-8494-63380380f11c.png)

R1 here would just be the rank of our empirical observation in the combined distribution, or the averaged rank if it's a part of a cluster.  Normally you repeat the calculation with R2, which would be the sum of all the ranks of the null distribution.  However, since R2 is just (N(N + 1))/2, we could easily calculate that directly.  You use the smaller of the two U values to calculate the effect size.

**Comment by danlwarren** [comment]
Wait, so is that just R1-1 over the number of replicates, assuming U1 is smaller than U2?  And it seems like U2 would basically always be greater than U1 unless there were a very, very tiny number of replicates, maybe even 1?  I should probably be doing this on a piece of paper instead of on github but there's a cat on me.

**Comment by danlwarren** [comment]
Actually I think we might want to modify this; as written it sort of maxes out at 0.5 where two distributions are equivalent and then 0 where they're non-overlapping in either direction.  If we just go with U1 (from the empirical observation) we can convert this to  something that scales from 0 (empirical observation is way lower than null distribution) to 1 (empirical observation is way higher than null distribution).  However at that point it just turns into the rank of our observation divided by the number of observations in our null distribution, which is basically just its percentile rank.

I started out with the idea that there had to be something smarter to do than just rank the empirical observation in the null distribution, but I believe that I have very cleverly just taken an existing statistic and collapsed it down to exactly that.  I just took the long way 'round to exactly where I started.
